backend/src/main/java/de/bund/digitalservice/ris/caselaw/domain/ScheduledPublicationService.java

```diff
@@ -53,7 +54,10 @@ public void handoverScheduledDocUnits() {
   private void handoverDocument(DocumentationUnit docUnit) {
     try {
       String email = docUnit.managementData().scheduledByEmail();
-      this.handoverService.handoverDocumentationUnitAsMail(docUnit.uuid(), email);
+      var result = this.handoverService.handoverDocumentationUnitAsMail(docUnit.uuid(), email);
+      if (!result.isSuccess()) {
+        throw new HandoverException(String.join(", ", result.statusMessages()));
+      }
```

Comment:
This is also what the /handover controller endpoint does. This way, we do not have to change the logic of the handover method, but still send out the error notification.